string command
geometry_msgs/Pose pose
float64[] q
